### PR TITLE
Extract createWfsFilter method to WfsFilterUtil

### PR DESCRIPTION
**Pull request by annarieger**
### Description:
Extracted `createWfsFilter` method from `WfsSearch` class and moved it to the new utility class `WfsFilterUtil` to make it reusable in other projects.

Please review @terrestris/devs

### CODE REVIEW

File: src/Util/WfsFilterUtil/WfsFilterUtil.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+   *   attributes that should be searched through.
+   * @param {Object} attributeDetails An object mapping feature types to an
+   *   array of attribute details.
+   * @return {OlFormatFilter} Filter to be used with WFS GetFeature requests.
+   * @private
+   */
+  static createWfsFilter(featureType, searchTerm, searchAttributes, attributeDetails) {
+
+    const attributes = searchAttributes[featureType];
+    const details = attributeDetails[featureType];
```
[the remark this answers is not in the document]
Comment (annarieger) [reply src/Util/WfsFilterUtil/WfsFilterUtil.js]: Nice catch, added it in 4a088bc

File: src/Field/WfsSearch/WfsSearch.jsx [hunk 2]
```diff
@@ -51,10 +51,30 @@ export class WfsSearch extends React.Component {
...
+     *        exactSearch: true
+     *      }
+     *    },
+     *    featType2: {...}
+     *   }
+     *   ```
      * @type {Object}
      */
-    attributeDetails: PropTypes.objectOf(PropTypes.arrayOf(
+    attributeDetails: PropTypes.objectOf(PropTypes.objectOf(
```
[the remark this answers is not in the document]
Comment (annarieger) [reply src/Field/WfsSearch/WfsSearch.jsx]: Thanks for friendly queue jumping @hwbllmnn :laughing: 

Just wondering, why the prop validation was ok previously, while the `attributeDetails` structure was always relied on nested objests and didn't include any arrays.. But now it definitely corresponds the expected object used in `createWfsFilter` method.

**Comment by annarieger** [comment]
@dnlkoch, your notes are addressed. Any objections to merge?

**Comment by dnlkoch** [comment]
LGTM! Please merge!

**Comment by annarieger** [comment]
Thanks, merging :pineapple:

**Pull request merged**